### PR TITLE
Update `_make_hgrid` docstring

### PR DESCRIPTION
**Pull request by ashjbarnes**
Closes issue #52

### CODE REVIEW

File: regional_mom6/regional_mom6.py [hunk 1]
```diff
@@ -510,14 +510,12 @@ def __init__(
...
-        spaced this way and latitude spaced with 0.1
-        cos(mean_latitude). This way, grids in the centre of the
-        domain are perfectly square, but decrease monatonically in
-        area away from the equator
+        spaced. This is very simple but suitable for small domains.
 
+        Note:
+            The intention is for the hgrid generation to be very flexible. In future updates, further options will built
+            into _make_grid(), probably wrapping some functions from the `Grid Gen repository <https://github.com/ESMG/gridtools>`_.
+            For now, you can customise the hgrid by simply overwriting the `hgrid.nc` file that's deposited in your `rundir` after initialising an `experiment`. To conserve the metadata, it might be easiest to read the file in, then modify the fields before re-saving.
```
Comment (navidcy) [inline regional_mom6/regional_mom6.py:518]: no need to promise things for the future, let's leave only what can happen now... when this future time come that we have implemented more things we will include it in the docs, right?

Comment (ashjbarnes) [reply regional_mom6/regional_mom6.py:518]: I think it's still important to tell users how to modify the respective grids. The intention of the package isn't to prescribe my two example grids, but leave things flexible. I agree making promises for the future is a bit odd but in this case I thought it gave an explanation of the intention of this function

Comment (navidcy) [reply regional_mom6/regional_mom6.py:518]: Keep that bit! But don't make promises for how this package will be in the future.

Comment (ashjbarnes) [reply regional_mom6/regional_mom6.py]: done

File: regional_mom6/regional_mom6.py [hunk 2]
```diff
@@ -510,14 +510,10 @@ def __init__(
     def _make_hgrid(self, gridtype):
         """Sets up hgrid based on users specification of
         domain. Default behaviour leaves latitude and longitude evenly
-        spaced.
-
-        If user specifies a resolution of 0.1 degrees, longitude is
-        spaced this way and latitude spaced with 0.1
-        cos(mean_latitude). This way, grids in the centre of the
-        domain are perfectly square, but decrease monatonically in
-        area away from the equator
```
Comment (navidcy) [inline regional_mom6/regional_mom6.py:519]: why was this info deleted?
this is what happens with `gridtype == "even_spacing"`?
will we loose this information or is it somewhere else?

**Comment by ashjbarnes** [comment]
this is the part of the docstring that you didn't like in issue #52, hence why I was deleting it to try and resolve this issue

File: regional_mom6/regional_mom6.py (hunk 2 above)
Comment (ashjbarnes) [reply regional_mom6/regional_mom6.py:519]: this is the part of the docstring that you didn't like in issue https://github.com/COSIMA/regional-mom6/issues/52, hence why I was deleting it to try and resolve this issue

Comment (navidcy) [reply regional_mom6/regional_mom6.py:519]: Hahahha

Comment (navidcy) [reply regional_mom6/regional_mom6.py:519]: Apparently I forgot

**Review by navidcy — APPROVED** [review]

**Pull request merged**